### PR TITLE
Feat/dapp 572

### PR DESCRIPTION
**Pull request by leodhb**
# Description
A new generic input range component

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [ ] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Test A

### CODE REVIEW

File: src/components/moleculars/sections/OfferSelectionSection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
```
Comment (NickNish09) [inline src/components/moleculars/sections/OfferSelectionSection/index.tsx:1]: Hmm what do you think about putting this component inside the page its going to be used?
We were trying to use the `components` directory only for static components that have no context with the application. In this case this component kinda knows about an offer.

What do you think?

**Review by thaisribon — COMMENTED** [review]
I don't know if i'm missing something, but I tried to test on storybook and the slider didn't work

File: src/components/moleculars/sections/OfferSelectionSection/index.tsx (hunk 1 above)
Comment (leodhb) [reply src/components/moleculars/sections/OfferSelectionSection/index.tsx]: Sure. I deleted the offer selection component. Everything can be done with the input range component without share context!

**Comment by leodhb** [comment]
> I don't know if i'm missing something, but I tried to test on storybook and the slider didn't work

@thaisribon We're using only the input range component. And move the range it's working now.

**Review by NickNish09 — APPROVED** [review]

**Pull request merged**